packages/eql-mapper/Cargo.toml

```
@@ -7,19 +7,22 @@ authors = [
   "James Sadler <james@cipherstash.com>",
   "Yuji Yokoo <yuji@cipherstash.com>",
   "Drew Thomas <drew@cipherstash.com>",
-  "Toby Hede <drew@cipherstash.com>"
+  "Toby Hede <drew@cipherstash.com>",
```

Looks like there's (an existing) copy/paste typo here:
```suggestion
  "Toby Hede <toby@cipherstash.com>",
```